sdk/include/xdk/rtltypes.h

```diff
@@ -2,6 +2,27 @@
  *                           Runtime Library Types                            *
  ******************************************************************************/
 
+$if (_WDMDDK_ || _WINNT_)
+#define FAST_FAIL_LEGACY_GS_VIOLATION           0
```

Comment:
Are you sure these defines actually exist in winnt.h of the PSDK?
Here's an example of search in official Win10 PSDK: no trace of these except for some values in the ks*** ASM headers: https://github.com/search?q=repo%3Atpn%2Fwinsdk-10%20FAST_FAIL&type=code